server/database/data/rawdaty.sql

```diff
@@ -0,0 +1,50 @@
+BEGIN;
+
+DROP TABLE IF EXISTS kindergarten,user, location, comments CASCADE;
+
+CREATE TABLE kindergarten(
+id SERIAL PRIMARY KEY,
+kindergarten_name VARCHAR(50)	NOT NULL,
+user_id	SERIAL	REFERENCES user(id),
+cover_image	TEXT NOT NULL,
+description	TEXT NOT NULL,
+location_id	SERIAL	REFERENCES location(id),
+phone_number INTEGER NOT NULL,
+latitude INTEGER,
+longitude INTEGER,
+min_price INTEGER,	
```

Comment:
use `FLOAT` here because they tend to say that it costs 99.99$ for example so we have to take this in consideration.